common/src/main/java/io/netty/util/concurrent/AutoScalingEventExecutorChooserFactory.java

```diff
@@ -213,8 +247,19 @@ private void tryScaleUpBy(int amount) {
             }
         }
 
+        @Override
+        public int activeExecutorCount() {
+            return state.get().activeChildrenCount;
+        }
+
+        @Override
+        public List<AutoScalingUtilizationMetric> executorUtilizations() {
+            return Collections.unmodifiableList(utilizationMetrics);
```

Comment:
Just wrap it in the constructor so we don't have to do this everything we call this method.